userAPI.js

```diff
@@ -0,0 +1,62 @@
+const Joi = require('joi');
+const express = require('express');
+const app = express();
+
+app.use(express.json());
+
+const users = [
```

Comment:
For now, it's okay, but it's better that you create different layers for your applications, and separate routing files, from others (services, controllers, models). In fact, if you read about MVC pattern, you will understand more about how to divide your application into layers.